templates/example_cell_clustering.ipynb

```diff
@@ -17,7 +17,9 @@
   {
```

Comment:
<p>Probably want to change "The following data directories/files will be created for pixel clustering:" to "The following data directories/files will be created for cell clustering:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>